frontend/src/UserList.js

```diff
@@ -99,112 +53,85 @@ export function UserList() {
         </Button>
       </SimpleGrid>
       <Divider />
-
       {data
-        ? data.user.affiliations.edges[0].node.organization.affiliatedUsers.edges.map(
-            edge => {
-              return (
-                <Box key={edge.node.id} width="100%">
-                  <PseudoBox
-                    display={{ md: 'flex' }}
-                    alignItems="center"
-                    onClick={() => {
-                      window.alert('clicked box')
-                    }}
-                    _hover={{ borderColor: 'gray.200', bg: 'gray.200' }}
-                    p="30px"
+        ? data.userList.edges.map((edge) => {
+            return (
+              <PseudoBox
+                key={edge.node.id}
+                role="userCard"
```

Comment:
The `role` attribute refers to an [ARIA role](https://www.w3.org/TR/using-aria/#aria-roles). There are fixed number of them defined in the standards and relied on by screen readers and other assistive technology. Most HTML elements have an implicit role (so a `<button />` has an implied `role=button`. 

PseudoBox here is probably a `<div>` by default, and the non-standard role of `userCard` is being added to it can be found by [Testing Library's `getAllByRole` function](https://testing-library.com/docs/dom-testing-library/api-queries#byrole) later. I don't know if this would have actively negative consequences for assitive tech users (given that that role isn't a real ARIA role), but it's not a good idea.

Breaking these components down smaller and [favouring composition](https://youtu.be/3XaXKiXtNjw?t=832) makes it easier to test. Smaller components can [spread props directly onto certain elements](https://github.com/canada-ca/tracker/blob/c66987d43e5bbe8c5a18931e2470f8a8ddc74865/frontend/src/EmailField.js#L27) they render, [allowing the use of `getByTestId`](https://github.com/canada-ca/tracker/blob/c66987d43e5bbe8c5a18931e2470f8a8ddc74865/frontend/src/__tests__/EmailField.test.js#L29) for really simple selections. This would be a better strategy to the way roles are being used here.

Reply:
@sleepycat Mike, thanks for the awesome feedback.  Do you think a "userCard component would be better here???